### PR TITLE
Solve the problem of dimension misuse.

### PR DESCRIPTION
**Pull request by darkliang**
We adjust the dimensions of all models to the order of n,c,s,h,w. And the new model code is **compatible** with the previous checkpoint.

### CODE REVIEW

File: opengait/modeling/losses/softmax.py [hunk 1]
```diff
@@ -14,31 +14,30 @@ def __init__(self, scale=2**4, label_smooth=True, eps=0.1, loss_term_weight=1.0,
...
     def forward(self, logits, labels):
         """
-            logits: [n, p, c]
+            logits: [n, c, p]
             labels: [n]
         """
-        logits = logits.permute(1, 0, 2).contiguous()  # [n, p, c] -> [p, n, c]
-        p, _, c = logits.size()
-        log_preds = F.log_softmax(logits * self.scale, dim=-1)  # [p, n, c]
+        # logits = logits.permute(1, 0, 2).contiguous()  # [n, p, c] -> [p, n, c]
```
Comment (darkliang) [inline opengait/modeling/losses/softmax.py:20]: useless comment

File: opengait/modeling/modules.py [hunk 2]
```diff
@@ -38,41 +38,35 @@ def __init__(self, forward_block):
...
-        n, s, c, h, w = x.size()
-        x = self.forward_block(x.view(-1, c, h, w), *args, **kwargs)
-        input_size = x.size()
-        output_size = [n, s] + [*input_size[1:]]
-        return x.view(*output_size)
+        n, c, s, h, w = x.size()
+        x = self.forward_block(x.transpose(
+            1, 2).view(-1, c, h, w), *args, **kwargs)
+        output_size = x.size()
+        return x.reshape(n, s, *output_size[1:]).transpose(1, 2)
```
Comment (ChaoFan996) [inline opengait/modeling/modules.py:48]: .contiguous()
